main.go

```diff
@@ -300,19 +321,51 @@ func doBootstrapAuthorize(config *menderConfig, opts *runOptionsType) error {
 	return nil
 }
```

Comment:
Maybe we can abstract this in separate function. This looks exact same as in `initDaemon` couple of lines below.